sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -78,3 +79,16 @@ def _is_cloud_event(event):
         return all([_ in event for _ in required]) and event['specversion'] == "1.0"
     except TypeError:
         return False
+
+def _deserialize_data(event, event_type):
```

Comment:
This smells like a method on the object (or likely a mixin since we have egevent/cloudevent) rather than a helper (Since if I blur my eyes event is self); there's also the "logical/conceptual locality" benefit.